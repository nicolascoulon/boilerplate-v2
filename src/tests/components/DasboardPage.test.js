import React from 'react';
import DashboardPage from '../../components/DashboardPage';
import { shallow } from 'enzyme';



test('should render DashboardPage corrctly', () => {
const wrapper = shallow(<DashboardPage />); 
expect(wrapper).toMatchSnapshot();  
});
