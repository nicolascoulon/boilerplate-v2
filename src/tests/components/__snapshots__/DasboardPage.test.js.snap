// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render DashboardPage corrctly 1`] = `<div />`;
